rust/windows-client/README.md

```diff
@@ -1,11 +1,48 @@
 # windows-client
 
-This crate houses the Firezone Windows client.
+This crate houses a Windows GUI client.
 
 ## Building
 
-Run `cargo build` in this directory.
+Given this Git repo is in the dir "firezone":
+
```

Comment:
```suggestion
```

For the other components we make the assumption the user is already in the component directory. I think it's safe to assume the same here.

Reply:
Okay. Is it okay to make this explicit and say "From this directory..."? I'll update the commands.

Reply:
Done in 0af598692e4a